### ISSUE TITLE
Panicked while starting the daemon server

### ISSUE THREAD

**Issue by eliseumds** [tkellogg/dura#7]
- MacOS Monterey 12.1 (21C52) (Intel)
- rustc 1.57.0 (f1edd0429 2021-11-29)

On commit f4d552da1f92f3d5ad85de483f2faa0a3d2efa65.

Steps executed:

- `brew install rustup`
- `rustup-init`
- `cargo install --path .`
- `RUST_BACKTRACE=1 dura serve`

And got:

```
pid: 62355
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/config.rs:63:14
stack backtrace:
   0: rust_begin_unwind
             at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/std/src/panicking.rs:517:5
   1: core::panicking::panic_fmt
             at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/core/src/panicking.rs:100:14
   2: core::result::unwrap_failed
             at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/core/src/result.rs:1616:5
   3: dura::config::Config::save
   4: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
   5: tokio::park::thread::CachedParkThread::block_on
   6: tokio::runtime::thread_pool::ThreadPool::block_on
   7: tokio::runtime::Runtime::block_on
   8: dura::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

**Comment by tkellogg** [comment]
Can you run `ls -latrh ~/.config/dura/`?

**Comment by bjorn3** [comment]
Looking at the source it seems the `~/.config/dura/` directory isn't created before trying to write the config file.

**Comment by darti** [comment]
Had the same issue.
PR: https://github.com/tkellogg/dura/pull/10

**Comment by tkellogg** [comment]
I just merged #10, @eliseumds would you mind updating & confirm that this is fixed?

**Comment by eliseumds** [comment]
@tkellogg @darti works, thanks a lot!
